data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Table.java

```diff
@@ -109,13 +109,20 @@ public String toDotString() {
 //				"label = \"" + getLabel() + "\"\n" +
 //				"shape = \"" + getShape() + "\"\n" +
 //				"];";
+
+		// struct1 [label=<
+		//<TABLE BORDER="0" CELLBORDER="1" CELLSPACING="0">
+		//<TR><TD>left</TD><TD PORT="f1">mid dle</TD><TD PORT="f2">right</TD></TR>
+		//</TABLE>>];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)